compiler/rustc_ast_pretty/src/pprust/state/fixup.rs

```diff
@@ -1,7 +1,9 @@
 use rustc_ast::Expr;
 use rustc_ast::util::{classify, parser};
 
-#[derive(Copy, Clone, Debug)]
+// The default amount of fixing is minimal fixing. Fixups should be turned on
+// in a targeted fashion where needed.
+#[derive(Copy, Clone, Debug, Default)]
```

Comment:
It might be better to explicitly comment that the default values for these fields should be `false`.